common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -144,7 +148,7 @@ private static boolean isWindows() {
     }
 
     private static boolean isOSX() {
-        return OSNAME.startsWith("macosx") || OSNAME.startsWith("osx");
+        return OSNAME.startsWith("osx");
```

Comment:
Note: the  OSDetector normalizes osx to always be 'osx'. This is one of the reasons I pulled it in!